packages/xrpl/src/client/index.ts

```diff
@@ -666,7 +667,57 @@ class Client extends EventEmitter<EventTypes> {
       promises.push(checkAccountDeleteBlockers(this, tx))
     }
 
-    return Promise.all(promises).then(() => tx)
+    // further manipulation of tx_ uses non-SubmittableTransaction types, hence we need a typecast to any
```

Comment:
If you have ideas for implementing this in a more clean fashion (fewer `eslint-disable` commands), please let me know